base/Dockerfile.deb9

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM azul/zulu-openjdk-debian:8u242
+FROM azul/zulu-openjdk-debian:8u262
```

Comment:
Thank you for this fix. I was very bothered with this when I first saw that we are using a 6 month old base image, but then it fell off my radar.